Gems/AtomLyIntegration/CommonFeatures/Code/Source/Material/EditorMaterialComponentSlot.cpp

```diff
@@ -130,6 +130,7 @@ namespace AZ
                     ->Method("OpenMaterialEditor", &EditorMaterialComponentSlot::OpenMaterialEditor)
                     ->Method("OpenMaterialInspector", &EditorMaterialComponentSlot::OpenMaterialInspector)
                     ->Method("OpenUvNameMapInspector", &EditorMaterialComponentSlot::OpenUvNameMapInspector)
+                    ->Method("ExportMaterial", &EditorMaterialComponentSlot::ExportMaterial)
```

Comment:
Is it possible for BehaviorContext to provide a default argument for the exportPath?

Reply:
I don't believe so, not without finding another function with a different signature